vsts/pipelines/_buildParallel.yml

```diff
@@ -0,0 +1,243 @@
+  displayName: 'Component Detection - OSS Compliance'
+  inputs:
+    ignoreDirectories: '$(Build.SourcesDirectory)/tests'
+
+- task: Docker@1
+  displayName: Container registry login
+  inputs:
+    command: login
+    azureSubscriptionEndpoint: ${{ parameters.ascName }}
+    azureContainerRegistry: ${{ parameters.acrName }}
+
+- task: ShellScript@2
+  displayName: 'Build build image'
+  inputs:
+    scriptPath: ./build/build-buildimages.sh
+  condition: and(succeeded(), eq(variables['BuildBuildImages'], 'true'))
+
+- task: ShellScript@2
+  displayName: 'Build runtime images'
+  inputs:
+    scriptPath: ./build/build-runtimeimages.sh
+  condition: and(succeeded(), eq(variables['BuildRuntimeImages'], 'true'))
+
+- task: ShellScript@2
+  displayName: 'Test script generator'
+  inputs:
+    scriptPath: ./build/test-buildscriptgenerator.sh
+  condition: and(succeeded(), eq(variables['TestBuildImages'], 'true'))
+
+- task: ShellScript@2
+  displayName: 'Test startup script generators'
+  inputs:
+    scriptPath: ./build/test-startupscriptgenerators.sh
+  condition: and(succeeded(), eq(variables['TestRuntimeImages'], 'true'))
+
+- task: ShellScript@2
+  displayName: 'Test build images'
+  inputs:
+    scriptPath: ./build/test-buildimages.sh
+    args: skipBuildingImages
+  condition: and(succeeded(), eq(variables['TestBuildImages'], 'true'))
+
+- task: ShellScript@2
+  displayName: 'Test runtime images'
+  inputs:
+    scriptPath: ./build/test-runtimeimages.sh
+    args: skipBuildingImages
+  condition: and(succeeded(), eq(variables['TestRuntimeImages'], 'true'))
+
+- task: DownloadBuildArtifacts@0
+  displayName: 'Download Build Artifacts'
+  inputs:
+    artifactName: drop
+  condition: and(succeeded(), eq(variables['TestIntegration'], 'true'))
+
+- task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
+  displayName: 'Shell Script'
+  inputs:
+    type: InlineScript
+    script: |
+     echo 'Hello World'
+     ls -l $(System.ArtifactsDirectory)
+     ls - l $(System.ArtifactsDirectory)/drop
+     ls -l $(System.ArtifactsDirectory)/drop/images
+  condition: and(succeeded(), eq(variables['TestIntegration'], 'true'))
+
+- task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
+  displayName: 'Pull and Retag recently built oryx build image'
+  inputs:
+    type: FilePath
+    scriptPath: ./vsts/scripts/pullandretag.sh
+    args: $(System.ArtifactsDirectory)/drop/images/build-images-acr.txt
+  condition: and(succeeded(), eq(variables['TestIntegration'], 'true'))
+
+- task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
+  displayName: 'Pull and Retag recently built oryx runtime images'
+  inputs:
+    type: FilePath
+    scriptPath: ./vsts/scripts/pullandretag.sh
+    args: $(System.ArtifactsDirectory)/drop/images/runtime-images-acr.txt
+  condition: and(succeeded(), eq(variables['TestIntegration'], 'true'))
+
+- task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
+  displayName: 'Creating oryxtests images to mock appservice environment'
```

Comment:
What is this for? I ask this because our `build-buildimages.sh` does build this image too, so wondering...